src/styles/container.scss

```
@@ -0,0 +1,41 @@
+$snake-width: 300px;
+$screen-medium: 600px;
+$screen-large: 900px;
+
+.container {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(1, minmax($snake-width, 1fr));
+
+  &__block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+    min-width: $snake-width;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+
+    color: rgb(255, 255, 255);
+
+    @each $name, $color in $colors {
+      &--#{$name} {
+        background-color: $color;
+      }
+    }
+  }
+}
+
+@media (min-width: $screen-medium) {
+  .container {
+    grid-template-columns: repeat(2, minmax($snake-width, 1fr));
+  }
+}
+
+@media (min-width: $screen-large) {
+  .container {
+    grid-template-columns: repeat(3, minmax($snake-width, 1fr));
+  }
+}
```

You could write @media in css selector (cool feature of SASS)

```suggestion
  .container {
@media (min-width: $screen-large) {
    grid-template-columns: repeat(3, minmax($snake-width, 1fr));
  }
}
```

---

Also, you could just change css variable, like we do in the lesson